src/visitor/mssql.rs

```diff
@@ -344,6 +345,21 @@ impl<'a> Visitor<'a> for Mssql<'a> {
 
         Ok(())
     }
+
+    fn visit_average(&mut self, avg: Average<'a>) -> visitor::Result {
+        self.write("AVG")?;
+
+        self.surround_with("(", ")", |ref mut s| {
+            s.write("CONVERT")?;
+
+            s.surround_with("(", ")", |ref mut s| {
+                s.write("DECIMAL(32,16),")?;
+                s.visit_column(avg.column)
+            })
+        })?;
```

Comment:
nitpick: it would be nice to have a small comment explaining why we can't write the simple version here